modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -195,16 +197,8 @@ cannot find symbol
 [ERROR]   location: variable streamingHelper of type org.mule.runtime.extension.api.runtime.streaming.StreamingHelper
 ----
 
-To fix this error, add the `mule-sdk-api` dependency to the pom.xml:
+The `org.mule.sdk.api` package is an *Experimental* feature still under active development. It shouldn't be used unless explicitly instructed by a non-experimental feature documentation page. Being experimental, Mulesoft does not guarantee support or backwards compatibility on any class defined in said package.
```

Comment:
```suggestion
The `org.mule.sdk.api` package is an experimental feature still under active development. Do not use this package unless it is explicitly instructed by a non-experimental feature documentation page. Since this package is experimental, MuleSoft does not guarantee support or backward compatibility on any class defined in the package.
```